install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
+    curl -s -A "$CURL_USER_AGENT" -f "$URL.sig" > /var/tmp/sandstorm-$BUILD.tar.xz.sig
+    curl -s -A "$CURL_USER_AGENT" -f "https://dl.sandstorm.io/keyring.gpg" > /var/tmp/sandstorm-keyring.gpg
+
+    if which gpg > /dev/null; then
+      if gpg --no-default-keyring --keyring /var/tmp/sandstorm-keyring.gpg --status-fd 1 \
```

Comment:
Oh, if an attacker could spoof both the keyring and the signature, then they could make the content read `\n[GNUPG:] VALIDSIG 160D2D577518B58D94C9800B63F227499DA8CCBD` and sign with a different key, which would defeat this.

I guess we're assuming that HTTPS is protecting the keyring?  Maybe we should just inline the keyring into the installer script, since you're already trusting that?